README.md

```diff
@@ -96,6 +98,49 @@ which will enable you to call the container like:
 sdcli go test
 ```
 
+## For Shells Other than `bash`
+
+In fish shell, you create a `~/.config/fish/functions/sdcli.fish` file with 755 permissions having contents:
+
+```bash
+function sdcli
+  set cwd (pwd -L)
+  set gopath "$GOPATH"
+  if test -z "$gopath"
```

Comment:
are we testing to see if $gopath is set? if so, I think we need to use the -n flag.